### PR TITLE
Added minimal documentation for _get_configuration_warning method

### PR DESCRIPTION
**Pull request by hasahmed**

### CODE REVIEW

File: doc/classes/Node.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
 			<return type="String">
 			</return>
 			<description>
+				The return value of this method is displayed as a warning in the `Scene Dock` if the script that overrides it is a `tool` script.
```
Comment (reduz) [inline doc/classes/Node.xml]: I would also add that returning an empty string produces no warning.

**Review by reduz — CHANGES_REQUESTED** [review]

File: doc/classes/Node.xml (hunk 1 above)
Comment (akien-mga) [reply doc/classes/Node.xml]: Note also that this is wrong formatting for inline code. It should be done with `[code]tool[/code]`. I don't think the "Scene Dock" needs special formatting, it's not a keyword. I would simply write it as "Scene dock".

**Comment by hasahmed** [comment]
Completed requested changes

File: doc/classes/Node.xml [hunk 2]
```diff
@@ -42,6 +42,8 @@
 			<return type="String">
 			</return>
 			<description>
+                The string returned from this method is displayed as a warning in the "Scene Dock" if the script that overrides it is a [code]tool[/code] script.
```
Comment (akien-mga) [inline doc/classes/Node.xml]: Please use tabs for indentation.

**Comment by hasahmed** [comment]
Changed from spaces to tabs.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
I just noticed now that this PR was made and merged in the `3.1` branch. Please always make PRs in the `master` branch in the future, the `3.1` branch is only for manually cherry-picked changes from the `master` branch.

**Pull request merged**